torax/_src/transport_model/combined.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2024 DeepMind Technologies Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The CombinedTransportModel class.
+
+A class for combining transport models.
+"""
+
+import dataclasses
+from typing import Sequence
+
+import jax
+import chex
+from jax import numpy as jnp
+
+from torax._src import array_typing
+from torax._src import state
+from torax._src.config import runtime_params_slice
+from torax._src.geometry import geometry
+from torax._src.pedestal_model import pedestal_model as pedestal_model_lib
+from torax._src.transport_model import runtime_params as runtime_params_lib
+from torax._src.transport_model import transport_model
+
+
+@chex.dataclass(frozen=True)
+class DynamicRuntimeParams(runtime_params_lib.DynamicRuntimeParams):
+  transport_model_params: Sequence[runtime_params_lib.DynamicRuntimeParams]
```

Comment:
Do we need more strict typing here?

Reply:
I didn't know whether `Sequence[Union[QLKNNDynamicRuntimeParams, BohmgyrobohmDynamicRuntimeParams, ...]]` was 'more proper' or whether it's ok to say it can be a sequence of any transport model dynamic runtime params.

I think for the moment letting it be any is ok, because otherwise we'd be imagining a use case that doesn't exist yet (e.g. what if there's a transport model that isn't compatible with Combined?)